Slack&whatsapp-notifier.py

```diff
@@ -41,8 +23,6 @@ def get_db_connection():
     except mysql.connector.Error as err:
         print(f"Error: {err}")

```

Comment:
add the errors to a log file as this will be running as schedules, so no one can view the error in real time.